app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -35,39 +34,24 @@ public LoginActivitySteps(LoginActivity activityClass) {
     }
 
     @Given("^I have a LoginActivity")
-    public void I_have_a_LoginActivity() {
+    public void i_have_a_LoginActivity() {
```

Comment:
Can you change these method names so they're consistent with Java conventions? So this would become `iHaveALogInActivity()`